MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -33,6 +35,14 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		}
 	}
 
+	func display(_ viewModel: FeedErrorViewModel) {
+		guard let message = viewModel.message else {
+			errorView?.hideMessage()
+			return
+		}
+		errorView?.show(message: message)
```

Comment:
`if/else` is more idiomatic and easier to read in this context.

For example: "If there's an error message, show it. Else, hide any existing message."

Instead of: "guard against a `nil` error message. Else, hide the current message and return. Show the error message."
```suggestion
		if let message = viewModel.message {
			errorView?.show(message: message)
		} else {
			errorView?.hideMessage()
		}
```